test/functional/fixtures/regression/gh-2546/test.js

```diff
@@ -0,0 +1,82 @@
+const path     = require('path');
+const expect   = require('chai').expect;
+const { exec } = require('child_process');
+
+describe('[Regression](GH-2546)', function () {
+    this.timeout(60000);
+
+    it('Should fail on uncaught promise rejection when ignoreUncaughtErrors is false', function () {
```

Comment:
We have [`skip-js-errors` flag](https://devexpress.github.io/testcafe/documentation/using-testcafe/command-line-interface.html#-e---skip-js-errors). Why not to call this one `skip-uncaught-errors`? But I'm not sure that `uncaught-errors` is enough descriptive name? @VasilyStrelyaev